product-web-app/service/jobRolesService.ts

```diff
@@ -16,4 +17,13 @@ export default class JobRoleService {
       throw new Error('Error jobRoleService');
     }
   }
+
+  async createJobRoles(addJobRole: AddJobRole): Promise<Number> {
+    try {
+      const response = await axios.post(`${this.API_URL}/admin/job-roles`, addJobRole);
+      return response.status;
```

Comment:
why do you return here http status?
your frontend Controller code is ignoring this value.
Backend is sending text with id - and maybe it doesn't have to if it's ignored.